.gitignore

```diff
@@ -0,0 +1 @@
+.code-workspace.code-workspace
```

Comment:
This is to ignore my code workspace. You don't have to have it in the master gitignore, it wont break anything to be there tho